Etabs_Adapter/CRUD/Read/Bar.cs

```diff
@@ -148,41 +148,41 @@ private List<Bar> ReadBar(List<string> ids = null)
 
         /***************************************************/
 
-        public static BarRelease GetBarRelease(bool[] startRestraint, double[] startSpring, bool[] endRestraint, double[] endSpring)
+        public static BarRelease GetBarRelease(bool[] startRelease, double[] startSpring, bool[] endRelease, double[] endSpring)
         {
-            Constraint6DOF startRelease = new Constraint6DOF();
-
-            startRelease.TranslationX = startRestraint[0] == true ? DOFType.Free : DOFType.Fixed;
-            startRelease.TranslationY = startRestraint[1] == true ? DOFType.Free : DOFType.Fixed;
-            startRelease.TranslationZ = startRestraint[2] == true ? DOFType.Free : DOFType.Fixed;
-            startRelease.RotationX = startRestraint[3] == true ? DOFType.Free : DOFType.Fixed;
-            startRelease.RotationY = startRestraint[4] == true ? DOFType.Free : DOFType.Fixed;
-            startRelease.RotationZ = startRestraint[5] == true ? DOFType.Free : DOFType.Fixed;
-
-            startRelease.TranslationalStiffnessX = startSpring[0];
-            startRelease.TranslationalStiffnessY = startSpring[1];
-            startRelease.TranslationalStiffnessZ = startSpring[2];
-            startRelease.RotationalStiffnessX = startSpring[3];
-            startRelease.RotationalStiffnessY = startSpring[4];
-            startRelease.RotationalStiffnessZ = startSpring[5];
-
-            Constraint6DOF endRelease = new Constraint6DOF();
-
-            endRelease.TranslationX = endRestraint[0] == true ? DOFType.Free : DOFType.Fixed;
-            endRelease.TranslationY = endRestraint[1] == true ? DOFType.Free : DOFType.Fixed;
-            endRelease.TranslationZ = endRestraint[2] == true ? DOFType.Free : DOFType.Fixed;
-            endRelease.RotationX = endRestraint[3] == true ? DOFType.Free : DOFType.Fixed;
-            endRelease.RotationY = endRestraint[4] == true ? DOFType.Free : DOFType.Fixed;
-            endRelease.RotationZ = endRestraint[5] == true ? DOFType.Free : DOFType.Fixed;
-
-            endRelease.TranslationalStiffnessX = endSpring[0];
-            endRelease.TranslationalStiffnessY = endSpring[1];
-            endRelease.TranslationalStiffnessZ = endSpring[2];
-            endRelease.RotationalStiffnessX = endSpring[3];
-            endRelease.RotationalStiffnessY = endSpring[4];
-            endRelease.RotationalStiffnessZ = endSpring[5];
-
-            BarRelease barRelease = new BarRelease() { StartRelease = startRelease, EndRelease = endRelease };
+            Constraint6DOF bhStartRelease = new Constraint6DOF();
+
```

Comment:
Push uses XZY, pull uses XYZ- I am not sure which one is right! I think it is XZY.